app/src/main/java/org/greenstand/android/TreeTracker/api/ApiService.kt

```diff
@@ -27,9 +26,15 @@ interface ApiService {
     @PUT("devices/")
     suspend fun updateDevice(@Body deviceRequest: DeviceRequest): PostResult
 
+    @POST("planter_accounts")
+    suspend fun planterAccountData(
+        @Body planterAccountRequest: PlanterAccountRequest
+    ): Set<PlanterAccountData>
+
     companion object {
 
         const val ENDPOINT = BuildConfig.BASE_URL
+        const val PAYMENT_ENDPOINT = BuildConfig.PLANTER_ACCOUNT_URL
```

Comment:
When we have one unified api, we could remove `PAYMENT_ENDPOINT`. For now, I am assuming two separate endpoints.